src/Microsoft.AspNetCore.Razor.Tasks/RazorTagHelper.cs

```diff
@@ -47,6 +47,33 @@ protected override bool SkipTaskExecution()
             return false;
         }
 
+        protected override bool ValidateParameters()
+        {
+            if (!Directory.Exists(ProjectRoot))
+            {
+                Log.LogError("The specified project root directory {0} doesn't exist.", ProjectRoot);
+                return false;
+            }
+
+            if (Configuration.Length == 0)
+            {
+                Log.LogError("The project {0} must provide a value for {1}.", ProjectRoot, nameof(Configuration));
+                return false;
+            }
+
+            for (var i = 0; i < Extensions.Length; i++)
+            {
+                if (!EnsureRequiredMetadata(Extensions[i], Identity) ||
+                    !EnsureRequiredMetadata(Extensions[i], AssemblyName) ||
+                    !EnsureRequiredMetadata(Extensions[i], AssemblyFilePath))
+                {
+                    return false;
+                }
+            }
+
```

Comment:
We should also validate that the references are full-qualified paths. The build server requires this.

Reply:
Added.